config.yaml

```diff
@@ -37,6 +37,25 @@ options:
     type: string
     default: "yes"
     description: Set to no to allow downloading from an invalid https site.
+  plugins-force-reinstall:
+      type: boolean
+      default: False
+      description: |
```

Comment:
You want to use > instead of | in this and the following two description fields. The | marker indicates preformatted text, but these three will render better in the store using the > flowed text YAML marker.